src/launchpad/actions/expressions.js

```diff
@@ -0,0 +1,57 @@
+const constants = require("../constants");
+const { generateKey } = require("../utils/utils");
+
+function evaluateInput(input) {
+  return async function ({dispatch, client}) {
+    if (!client) {
```

Comment:
meh, i don't think we need this :/

We could handle that better somewhere else